src/AndrewBalint/TaskUtilityMethods.java

```diff
@@ -0,0 +1,85 @@
+package AndrewBalint;
+
+public class TaskUtilityMethods {
+
+
+    public static void main(String[] args) {
+
+        System.out.println(identify(11));
+
+        System.out.println(NDivide(99,25));
+
+        FINRA();
+
+    }
+
+    public static String identify(int num){
```

Comment:
Greate work.
But I belive your method "identify" should work with all numbers. For now it working only with 0 and positive numbers.  
Correct me if I wrong.